DataFormats/CTPPSDigi/interface/HPTDCErrorFlags.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CTPPSDigi_HPTDCErrorFlags_h
+#define CTPPSDigi_HPTDCErrorFlags_h
+
+#include <string>
+
+/** \class HPTDCErrorFlags
+ *
+ * HPTDC error flags container
+ *  
+ * \author Seyed Mohsen Etesami
+ * \author Laurent Forthomme
+ * July 2016
+ */
+
+// namespace HPTDCErrors { 
+struct HPTDCErrorFlags {
+HPTDCErrorFlags(unsigned short flags=0): error_flags(flags) {;}
```

Comment:
the unnecessary semicolon in ``` {;}``` is not in 91X.
Why do we need it in the backport?